### PR TITLE
feat(sessions): document redis sessions config

### PR DESCRIPTION
**Pull request by abailleulfc**

### CODE REVIEW

File: source/docs/advanced/production-ready/overview.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ In this section, you will learn advanced information about the different things
 - [Optimize your media](./media-middleware.html)
 - [Configure multiple stores](./multistore.html)
 - [Setup and use logging](./logging.html)
+- [Setup sessions](./sessions.html)
```
Comment (real34) [inline source/docs/advanced/production-ready/overview.md:11]: **issue:** the page is not available in the navigation menu

**suggestion:** add it in the config.yml menu list too

File: source/docs/advanced/production-ready/sessions.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    "127.0.0.1",
+  port:
+    process.env.FRONT_COMMERCE_CLOUD_REDIS_SESSIONS_PORT ||
+    process.env.FRONT_COMMERCE_REDIS_SESSIONS_PORT ||
+    6379,
+  db: process.env.FRONT_COMMERCE_REDIS_SESSIONS_DB || 2,
+});
+
+module.exports = {
+  // see https://developers.front-commerce.com/docs/reference/configurations.html#config-sessions-js
```
Comment (real34) [inline source/docs/advanced/production-ready/sessions.md:31]: :exploding_head: 
```suggestion
  // see https://developers.front-commerce.com/docs/advanced/production-ready/sessions.html
```

**Review by real34 — CHANGES_REQUESTED** [review]

**Comment by mo-fc** [comment]
other that pierre comments it looks good

File: source/docs/advanced/production-ready/overview.md (hunk 1 above)
Comment (abailleulfc) [reply source/docs/advanced/production-ready/overview.md:11]: added

File: source/docs/advanced/production-ready/sessions.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+---
+id: sessions
+title: Setup sessions
+---
+
+By default the user sessions are stored on disk which may not work with cloud deployments.
```
[the remark this answers is not in the document]
Comment (abailleulfc) [reply source/docs/advanced/production-ready/sessions.md]: added reference in configurations.md

File: source/docs/advanced/production-ready/sessions.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+# Configure sessions via redis
+
+You can handle user session using redis by configuring it in `my-module/config/sessions.js`.
+
+```
```
[the remark this answers is not in the document]
Comment (abailleulfc) [reply source/docs/advanced/production-ready/sessions.md]: applied

File: source/docs/advanced/production-ready/sessions.md (hunk 2 above)
Comment (abailleulfc) [reply source/docs/advanced/production-ready/sessions.md]: move to seeception :smile:

**Review by real34 — APPROVED** [review]
Thanks :+1:

**Pull request merged**